source/extensions/filters/network/thrift_proxy/router/router_impl.h

```diff
@@ -164,7 +164,13 @@ class RouteMatcher {
   COUNTER(route_missing)                                                                           \
   COUNTER(unknown_cluster)                                                                         \
   COUNTER(upstream_rq_maintenance_mode)                                                            \
-  COUNTER(no_healthy_upstream)
+  COUNTER(no_healthy_upstream)                                                                     \
+  COUNTER(request_call)                                                                            \
+  COUNTER(request_oneway)                                                                          \
+  COUNTER(request_invalid_type)                                                                    \
+  COUNTER(response_reply)                                                                          \
+  COUNTER(response_exception)                                                                      \
+  COUNTER(response_invalid_type)
```

Comment:
I don't think these are necessary? You're adding these metrics to the router stats but you only ever increment them on cluster stats.